src/content/developers/tutorials/hello-world-smart-contract-fullstack/index.md

```diff
@@ -0,0 +1,1548 @@
+
+```
+package name: (hello-world)
+version: (1.0.0)
+description: hello world smart contract
+entry point: (index.js)
+test command:
+git repository:
+keywords:
+author:
+license: (ISC)
+
+About to write to /Users/.../.../.../hello-world/package.json:
+
+{
+   "name": "hello-world",
+   "version": "1.0.0",
+   "description": "hello world smart contract",
+   "main": "index.js",
+   "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1"
+   },
+   "author": "",
+   "license": "ISC"
+}
+```
+
+Approve the package.json and we’re good to go!
+
+### Step 7: Download Hardhat {#step-7-download-hardhat}
+
+Hardhat is a development environment to compile, deploy, test, and debug your Ethereum software. It helps developers when building smart contracts and dApps locally before deploying to the live chain.
+
+Inside our `hello-world` project run:
+
+```
+npm install --save-dev hardhat
+```
+
+Check out this page for more details on [installation instructions](https://hardhat.org/getting-started/#overview).
+
+### Step 8: Create Hardhat project {#step-8-create-hardhat-project}
+
+Inside our `hello-world` project folder, run:
+
+```
+npx hardhat
+```
+
+You should then see a welcome message and option to select what you want to do. Select “create an empty hardhat.config.js”:
+
+```
+888    888                      888 888               888
+888    888                      888 888               888
+888    888                      888 888               888
+8888888888  8888b.  888d888 .d88888 88888b.   8888b.  888888
+888    888     "88b 888P"  d88" 888 888 "88b     "88b 888
+888    888 .d888888 888    888  888 888  888 .d888888 888
+888    888 888  888 888    Y88b 888 888  888 888  888 Y88b.
+888    888 "Y888888 888     "Y88888 888  888 "Y888888  "Y888
+
+👷 Welcome to Hardhat v2.0.11 👷‍
+
+What do you want to do? …
+Create a sample project
+❯ Create an empty hardhat.config.js
+Quit
+```
+
+This will generate a `hardhat.config.js` file for us, which is where we’ll specify all of the setup for our project (on step 13).
+
+### Step 9: Add project folders {#step-9-add-project-folders}
+
+To keep our project organized we’ll create two new folders. Navigate to the root directory of your `hello-world` project in your command line and type:
+
+```
+mkdir contracts
+mkdir scripts
+```
+
+- `contracts/` is where we’ll keep our hello world smart contract code file
+- `scripts/` is where we’ll keep scripts to deploy and interact with our contract
+
+### Step 10: Write our contract {#step-10-write-our-contract}
+
+You might be asking yourself, when the heck are we going to write code?? Well, here we are, on Step 10.
+
+Open up the hello-world project in your favorite editor (we like [VSCode](https://code.visualstudio.com)). Smart contracts are written in a language called Solidity which is what we will use to write our HelloWorld.sol smart contract.‌
+
+1. Navigate to the “contracts” folder and create a new file called `HelloWorld.sol`
```

Comment:
We keep flip-flopping between using backticks and using quotes. Using backticks for consistency for now. Also, we handhold teaching people in making two directories but assume they can create files.

```suggestion
1. Navigate to the `contracts` folder and create a new file called `HelloWorld.sol`
```